frontend/src/components/common/ConfirmationMessage.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { ADMIN_PAGE } from "../../constants/Routes";
+import { ReactComponent as EnvelopePaperIllustration } from "../../assets/illustrations/envelope-paper.svg";
+import MessageContainer from "./MessageContainer";
+
+const ConfirmationMessage = (): React.ReactElement => {
+  return (
+    <MessageContainer
+      illustration={
+        <EnvelopePaperIllustration
+          style={{ paddingBottom: "1.5em", margin: "auto" }}
```

Comment:
shouldn't these spacing attributes always be necessary? if so, should we refactor into `MessageContainer` so that it can be used in the future?